ft-sdk/src/auth/mod.rs

```diff
@@ -17,7 +17,8 @@ pub struct UserId(pub i64);
 #[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Default)]
 #[serde(default)]
 pub struct ProviderData {
-    pub identity: String,
+    // identity can be None which indicates that this data is imported from somewhere
+    pub identity: Option<String>,
```

Comment:
We have only made fastn_user’s identity column nullable, for now identity is still required for provider data. I am not sure as to the reason, check the discussions I have shared recently. If you still think identity should be made nullable let me know. Also if possible figure out why it should not be nullable based on my past discussions and add a comment here about it.

Reply:
You're right, we don't need this. I was trying to reuse some functions of the auth provider to get UserData and thought I would need the identity to be optional.